docs/metrics/customizing-the-sdk/README.md

```diff
@@ -224,6 +224,51 @@ default boundaries. This requires the use of
         new ExplicitBucketHistogramConfiguration { Boundaries = Array.Empty<double>() })
 ```
 
+#### Determining Explicit Buckets for Histograms
+
+In OpenTelemetry .NET, histograms can be customized using both
+the View API and the Advice API. The actual explicit buckets used by
+the SDK are determined based on the following factors:
+
+1. The explicit buckets provided by the instrumentation owner via the Advice API.
+2. The explicit buckets defined through the View API during SDK initialization.
+3. The default buckets provided by the SDK.
+
+Here’s how the SDK determines which explicit buckets to use:
+
+1. **View API**: If custom buckets are defined via the View API, these
+take precedence over any buckets provided by the Advice API or SDK defaults.
+2. **Advice API**: If the View API does not specify custom buckets, the SDK
+uses the explicit buckets provided by the Advice API.
+3. **SDK Defaults**: If neither the View API nor the Advice API specifies
+custom buckets, the SDK applies its default configuration.
+
+Additionally, if an exponential histogram is defined using the View API,
+the explicit bucket boundaries provided by the Advice API will be ignored,
+as exponential histograms take precedence.
```

Comment:
Want to make sure I understand this - if the user leveraged the View API to choose a subset of the attributes, and there are no explicit buckets provided - would the buckets provided by the Hint API take effect or not?

Reply:
Yes, the buckets provided by the Advice (Hint) API would take effect in this case. Here's what I understand:

1. Exponential Histogram with View API: If the user defines an exponential histogram via the View API, it takes full precedence, and any explicit bucket boundaries provided by the Advice API are ignored.

2. No Explicit Buckets Provided by View API: If the View API is used but does not define explicit bucket boundaries (e.g., it only filters attributes or configures other aspects), then:

- The SDK will fall back to the explicit buckets from the Advice (Hint) API if they are available.

- If no buckets are defined in the Advice API either, then the SDK defaults apply.

@cijothomas Could you please verify if these are correct?